lib/zendesk_apps_support/validations/translations.rb

```diff
@@ -47,6 +47,15 @@ def json_error(file)
           ValidationError.new('translation.not_json', file: file.relative_path, errors: e)
         end
 
+        def required_keys(file)
+          return nil unless file.to_s == 'translations/en.json'
+
+          json = JSON.parse(file.read)
+          return nil if json['app']['name']
+
+          ValidationError.new('translation.missing_required_key', file: file.relative_path, missing_key: 'app.name')
```

Comment:
this is weird...The validation of translation keys already happen here: https://github.com/zendesk/apps_approval/blob/master/lib/apps_approval/validators/translations_validator.rb#L40